build-scripts/fetch-binaries.sh

```diff
@@ -28,4 +28,16 @@ echo $KUBE_VERSION > $KUBE_SNAP_BINS/version
   mkdir -p cni
   curl -LO https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-$KUBE_ARCH-${CNI_VERSION}.tgz
   tar -zxvf cni-plugins-$KUBE_ARCH-${CNI_VERSION}.tgz -C cni
+
+  ISTIO_ERSION=$(echo $ISTIO_VERSION | sed 's/v//g')
```

Comment:
ISTIO_ERSION, is that a typo?

Reply:
Nope :) Version is "v1.0.0" but we need only the "1.0.0" for the url below. So that is the ISTIO_VERSION without the V

Reply:
I think Adam means the variable name `ISTIO_ERSION`